### DISCUSSION TITLE
Resetting session for a single user

### DISCUSSION THREAD

**Discussion by bo-qeye** [neutrinolabs/xrdp#3253] [Q&A]
Server: Rocky Linux 8.10, xrdp 0.10.1
Client: Microsoft Remote Desktop for Mac

I have multiple users using xrdp on the same server. One user locked up his session by spawning many windows. How can I reset that particular user's session only?

**Comment by jsorg71** [comment]
I usually kill the Xorg for that user.
ps aux | grep Xorg
look for pid for that user Xorg
kill <pid>
that should cleanup the session.

**Reply by bo-qeye** [reply]
Hmm, Rocky Linux doesn't use Xorg processes, it seems...

**Reply by akarl10** [reply]
I usually have better success with `loginctl kill-user <username>`
Since only one graphical session is possible this should be what you need.
else with `loginctl list-sessions` you can lookup all session and kill only the specific session with `loginctl kill-session <session id>` (it should be the one with class "user", no tty and no seat)

**Comment by matt335672** [comment]
By default, Rocky/Alma/CentOS/RHEL uses Xvnc for the X server. Try `ps aux | grep Xorg` instead.
